src/lib.rs

```diff
@@ -90,6 +90,9 @@ pub enum Exception {
     /// Allow reading an environment variable.
     Environment(String),
 
+    /// Allow reading **all** environment variables.
+    FullEnvironment,
```

Comment:
Not sure if `FullEnvironment` is the perfect name, but I prefer it over `AnyEnvironment` and `AllEnvironment`.

Reply:
I can't think of anything better...